packages/block-library/src/template-part/index.php

```diff
@@ -51,8 +51,12 @@ function render_block_core_template_part( $attributes ) {
 		}
 	}
 
-	if ( is_null( $content ) ) {
-		return 'Template Part Not Found';
+	if ( is_null( $content ) && is_user_logged_in() ) {
+		return sprintf(
+			/* translators: %s: template part slug. */
```

Comment:
Same as above, maybe "Template" should be capitalized?
```suggestion
			/* translators: %s: Template part slug. */
```